helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -52,6 +50,7 @@ import org.ossreviewtoolkit.model.Provenance
 import org.ossreviewtoolkit.model.RepositoryProvenance
 import org.ossreviewtoolkit.model.Severity
 import org.ossreviewtoolkit.model.TextLocation
+import org.ossreviewtoolkit.utils.common.FileMatcher
```

Comment:
IMO the commit message should not refer to "Ant's path matcher", but just to our "FileMatcher".